src/ActionModal/components/ActionModal.vue

```diff
@@ -85,8 +85,9 @@
           </TmFormGroup>
           <TableInvoice
             :amount="Number(subTotal)"
-            :estimated-fee="estimatedFee"
+            :estimated-fee="alternativeEstimatedFee || estimatedFee"
             :bond-denom="getDenom"
+            :fee-denom="alternativeFeeDenom || getDenom"
```

Comment:
instead make fee-denom optional?

Reply:
actually I just noticed this wasn't needed, as it was already optional